third_party/blink/renderer/farbling/brave_session_cache.cc

```diff
@@ -325,11 +393,13 @@ bool BraveSessionCache::AllowFontFamily(
   return true;
 }
 
-FarblingPRNG BraveSessionCache::MakePseudoRandomGenerator() {
-  uint64_t seed = *reinterpret_cast<uint64_t*>(domain_key_);
+FarblingPRNG BraveSessionCache::MakePseudoRandomGenerator(FarbleKey key) {
+  uint64_t seed = *reinterpret_cast<uint64_t*>(domain_key_) ^ (unsigned int)key;
```

Comment:
I'm not understanding what's going on with this FarbleKey, but also don't use c-style casts

Reply:
and don't use implicitly typed enums `enum FarbleKey : uint64_t`

Reply:
Fixed. I also added a comment to hopefully make it more clear.

Reply:
have you pushed the changes? Still looks the same to me

Reply:
it's best to let github auto-resolve when you push changes